journals/apps/journals/admin.py

```diff
@@ -22,10 +27,34 @@ class JournalAboutPageAdmin(admin.ModelAdmin):
 
 
 @admin.register(Journal)
-class JournalAdmin(admin.ModelAdmin):
+class JournalAdmin(DjangoObjectActions, admin.ModelAdmin):
+    """
+    Admin for Journal model.
+    """
+    change_actions = ('import_users',)
     fields = ('uuid', 'journalaboutpage', 'name', 'access_length', 'organization', 'video_course_ids')
     readonly_fields = ('uuid', 'journalaboutpage')
 
+    def get_urls(self):
+        my_urls = [
+            url(
+                r'(?P<journal_id>[0-9]+)/actions/import_users/',
+                JournalAccessView.as_view(),
```

Comment:
I think we need to wrap `JournalAccessView` around  [admin_view](https://docs.djangoproject.com/en/1.11/ref/contrib/admin/#django.contrib.admin.ModelAdmin.get_urls) to enforce admin permissions on it.

Reply:
If its only reason is to add admin permissions then I am already adding the permission in my view. I am tagging you there. Or is there anything else I am missing?